### ISSUE TITLE
Service worker manifest references non-existent inject_styles.js

### ISSUE THREAD

**Issue by illright** [sveltejs/sapper#1559]
**Describe the bug**
If you don't set `emitCss: true` in `rollup.config.js`, the `inject_styles.js` won't appear in the `__sapper__/dev/client` and the service worker will fail to fetch this file.

When observing this issue, I was perplexed by the fact that if you navigate to the URL of that file (`http://localhost/client/inject_styles.88de199d.js`) in the browser, it gets fetched successfully. However, if you try the same in incognito, with a service worker or `curl`, you'll see a 404 from Sapper because it treats this URL like a route.

**To Reproduce**
* Clone the Sapper template
* Install the dependencies
* Remove the `emitCss: true` option in `rollup.config.js` for the client
* Run the server in dev mode
* Visit `localhost:3000/`, then reload the page

**Expected behavior**
No errors should appear in the console.

**Information about your Sapper Installation:**

- The output of `npx envinfo --system --npmPackages svelte,sapper,rollup,webpack --binaries --browsers`
```
  System:
    OS: Linux 5.8 Manjaro Linux
    CPU: (8) x64 Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz
    Memory: 1.59 GB / 7.62 GB
    Container: Yes
    Shell: 3.1.2 - /usr/bin/fish
  Binaries:
    Node: 14.10.0 - /usr/bin/node
    Yarn: 1.22.5 - /usr/bin/yarn
    npm: 6.14.7 - /usr/bin/npm
  Browsers:
    Firefox: 80.0.1
```
- Your browser
**Google Chrome, Version 85.0.4183.102**
- Your hosting environment (i.e. Local, GCP/AWS/Azure, Vercel/Begin, etc...)
**Local**
- If it is an exported (`npm run export`) or dynamic application
**Dynamic**

**Severity**
Some styles on my website are badly broken after the update, I'm suspecting this has something to do with it.

**Comment by benmccann** [comment]
`inject_styles.js` should not be created without `emitCss: true`, so that's the correct behavior. The issue seems to be that `create_serviceworker_manifest` is for some reason putting this file in the manifest

**Comment by illright** [comment]
Then how can we explain that I can fetch this file with the browser if it doesn't exist?

**Comment by Conduitry** [comment]
This should be fixed in 0.28.10.
